tests/unittests/OperatorTest.cpp

```diff
@@ -2367,6 +2367,22 @@ TEST_P(Operator, NonSquarePaddingMaxPool) {
   EXPECT_TRUE(result.isEqual(result1));
 }
 
+TEST_P(Operator, Int8Poolavg) {
```

Comment:
Int8AvgPool (there was some renaming of maxpool and avgpool)

Reply:
renamed